core/arch/instr.h

```diff
@@ -1788,6 +1788,17 @@ DR_API
 bool
 instr_writes_to_exact_reg(instr_t *instr, reg_id_t reg, dr_opnd_query_flags_t flags);
 
+DR_API
```

Comment:
All new API functions need an entry in release.dox.

Reply:
Done